Marlin/src/pins/stm32f1/pins_CREALITY_V431.h

```diff
@@ -0,0 +1,43 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2020 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ * CREALITY v4.2.7 (STM32F103) board pin assignments
+ */
+
+#if NOT_TARGET(__STM32F1__)
+  #error "Oops! Select an STM32F1 board in 'Tools > Board.'"
+#endif
+
+#define BOARD_INFO_NAME      "Creality v4.3.1"
+#define DEFAULT_MACHINE_NAME "Ender 6"
+
+//
+// Steppers
+//
+#define X_STEP_PIN                        PB8
+#define X_DIR_PIN                         PB7
+
+#define Y_STEP_PIN                        PC2
+#define Y_DIR_PIN                         PB9
```

Comment:
You're going to get a bunch of redefined warnings unless you add some #ifndef's to `pins_CREALITY_V4.h`.

Reply:
I didn't have that issue, should be because pins_CREALITY_V4.h is included at the end of pins_CREALITY_V431.h. This is the same pattern used by pins_CREALITY_V427.h.

Reply:
Oops, you're correct. I read `pins_CREALITY_V4.h` incorrectly and those pins are already wrapped in #ifndefs. 🙂

https://github.com/MarlinFirmware/Marlin/blob/e82c7a309f0ac63a23ea997c6387ef713f3167d1/Marlin/src/pins/stm32f1/pins_CREALITY_V4.h#L86-L100